app/bundles/LeadBundle/EventListener/OwnerSubscriber.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
```

Comment:
Update the year for newly added files
```suggestion
 * @copyright   2019 Mautic Contributors. All rights reserved
```

Reply:
Fixed.

Reply:
*Note* I assumed the php-cs-fixer would handle this and hadn't even looked at it since my IDE collapses it by default. Maybe a PR could be opened up which manages that automatically.